db/db.go

```diff
@@ -0,0 +1,31 @@
+package db
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+	"practice/go-todo-list/config"
+)
+
+var db *sql.DB
+
+func InitDB() *sql.DB {
+	var err error
+
+	fmt.Println(config.Db().ConnString())
+	//jaadu := "postgres://localhost:5432/todo_db_test?sslmode=disable"
```

Comment:
`jaadu`? committed this? 😛 be sure to delete this things before committing the code

Reply:
:P Done.